tests/test_collider.py

```diff
@@ -0,0 +1,333 @@
+import taichi as ti
+import numpy as np
+import pytest
+from unittest.mock import patch, MagicMock
+
+# Create the mock before importing genesis modules
```

Comment:
Why do you need this instead of the initialising Genesis?

Reply:
My immediate goal is to be able to run _func_narrow_phase on Warp easily, to find out how long it takes to load from cache etc (does it load in milliseconds? 1 second? 3 seconds? Empirical question I feel). In order to try on warp, I want to minimize the scope of things I'd need to change from ti.field to the warp equivalent.

Reply:
Yeah, I'm not sure. That's why I've closed these for now. Will reponder their future a bit later.